.idea/vcs.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="IssueNavigationConfiguration">
-    <option name="links">
-      <list>
-        <IssueNavigationLink>
-          <option name="issueRegexp" value="FLINK\-\d+" />
```

Comment:
Is this file removed on purpose? We still have some Flink issues links in the code, and this helps link to Flink issues.

Reply:
@wuchong My mistake I thought it was introduced on my local branch.
I noticed the print statement while examining the Paimon connector, but if it was intentional I will just close this PR